docs/writing_modules.md

```diff
@@ -273,11 +274,53 @@ But if we reversed the order above:
 Then a request for `/example/one/two/three/four` would match `/one/...` last, and so would execute
 the `one()` handler function.
 
+## Outbound HTTP requests
+
+As the WASI specification is in the process of [adding support for Berkeley
+sockets][wasi-berkeley-sockets], WAGI enables _experimental_ outbound HTTP requests through the
+[`wasi-experimental-http`][wasi-experimental-http] library, which adds support
+for guest modules written in Rust and AssemblyScript to execute HTTP 1.1
+requests.
+Check the documentation and examples from the repository for complete programs
+that send HTTP requests, but here is a short example of a GET request in Rust:
+
+```rust
+use http;
+use wasi_experimental_http;
+
+#[no_mangle]
+pub extern "C" fn _start() {
+    let url = "https://api.brigade.sh/healthz".to_string();
+    let req = http::request::Builder::new().uri(&url).body(None).unwrap();
+    let res = wasi_experimental_http::request(req).expect("cannot make get request");
+
+    let str = std::str::from_utf8(&res.body()).unwrap().to_string();
+    println!("Content-Type: text/plain\n");
```

Comment:
Super nit: this should have `charset=utf-8` set

Reply:
Should it though?
I took it from the docs above.

Reply:
Probably, but no need to if you don't want to

Reply:
Opened #34 to track this.